addOns/scripts/src/main/javahelp/org/zaproxy/zap/extension/scripts/resources/help/contents/automation.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
+<TITLE>Scripts Automation Framework Support</TITLE>
+</HEAD>
+<BODY>
+	<H1>Scripts Automation Framework Support</H1>
+	This add-on supports the Automation Framework.
+
+	<H2>Job: script</H2>
+    The script job allows you to execute various actions with scripts.
+    Run standalone scripts is currently the only action, but more actions are planned (add/remove/enable/disable)
+    <pre>
+  - type: script
+    parameters:
+      action:                          # String: The executed action - available actions: run
+      scriptType:                      # String: The type of the script - available script type for action run: standalone
+      scriptName:                      # String: The name of the script
```

Comment:
Do we need the `script` prefix in both of these cases? 

Ideally I'd also like us to support a `path` param so that the script doesnt have to be pre configured in ZAP. If supplied then the script would be registered by ZAP when the action runs. However this can be added later.
If/when we do that we'll also need to support the engine name.

For future script types we'll need to support script parameters as well. I know they are not required now but its worth thinking ahead, esp as we could end up overloading the term`parameters`.
One option would be 
```
  parameters:
    action: run
    type: targetted
    name: 'My action'
    scriptParameters:
      target: http://www.example.com/
```

Thoughts anyone?

Reply:
Maybe we should move the discussion for future ScriptJob implementations [here](https://github.com/zaproxy/zaproxy/issues/7025)

Reply:
still `scriptType` and `scriptName` :/

Reply:
now adjusted also the docs